Client/Frontend/Login Management/LoginListViewController.swift

```diff
@@ -394,10 +439,38 @@ private class LoginCursorDataSource: NSObject, UITableViewDataSource {
 
     var cursor: Cursor<Login>?
 
+    var loginFaviconMap = [Login: [Favicon]]()
```

Comment:
Use the `GUID` here instead of the `Login` instance, and then you don't have to make its `Hashable` implementation lie.